src/main.js

```diff
@@ -0,0 +1,20 @@
+function daysInMonth(month, year) {
+  try {
+    if (!Number.isInteger(year) || month <= 0 || !Number.isInteger(month) || month > 12) {
+      throw new Error("Incorrect data: false date passed");
+    }
+        
+    return new Date(year, month, 0).getDate();
+
+  } catch(error) {
+    return error.message;
```

Comment:
you need to `throw an error` instead of `return message`

Reply:
done